### PR TITLE
Flesh out some of the FIXMEs

### PR DESCRIPTION
**Pull request by BjarniRunar**
(This includes @dkg 's current outstanding changes; if those get merged this will look much more reasonable. I'm very sorry if this is a crappy way to do this.)

Fill out some of the "FIXME" sections of the spec:

- Heading Capitalization-Tweaks
- Removed FIXME from Obscured Headers
- Removed FIXME regarding Enigmail protected-headers=v1 parameter
- Move "Message Rendering: Omitting..." into Message Interpretation
- New content:
   - Message Interpretation
   - Reverse-copying
   - Signature Invalidation
   - Reply/Forward Losing Subjects
   - Usability Impact of Reduced Metadata
   - Mailing List Header Modifications

### CODE REVIEW

File: draft-protected-headers.in.md [hunk 1]
```diff
@@ -399,6 +396,40 @@ A MUA SHOULD transform a Cryptographic Payload to include a Legacy Display part
...
+This document does not currently provide comprehensive recommendations on how to interpret Protected Headers. This is deliberate; research and development is still ongoing. We also recognize that the tolerance of different user groups for false positives (benign conditions misidentified as security risks), vs. their need for strong protections varies a great deal and different MUAs will take different approaches as a result.
+
+Some common approaches are discussed below.
+
+Reverse-Copying
+---------------
+
+One strategy for interpreting Protected Headers on an incoming message, is to simply ignore any Exposed Headers for which a Protected counterpart is available. This is often interpreted as a copy operation within the code which takes care of parsing the message.
+
+MUAs implementing this strategy should in pay special attention to any user facing headers (as defined above). If user-facing headers are among the Exposed Headers, but missing from the Protected Header section then the copy strategy actually implies deleting such Exposed Headers before presenting the message to the user.
```
Comment (juga0) [inline draft-protected-headers.in.md]: s/in pay special/pay special/

File: draft-protected-headers.in.md [hunk 2]
```diff
@@ -503,16 +535,14 @@ The impact of obscuring `From:`, `To:`, and `Cc:` headers has similar issues as
...
 
-(describe the issue, that some mailinglist softwares changes the `From:` line or `Reply-To:` in order to make sure that replies go to the list and not to the author.
-This is known as [header munging](https://www.unicom.com/pw/reply-to-harmful.html).
-Protected headers break this workaround.
-See https://cr.yp.to/proto/replyto.html how this should be fixed properly in a MUA)
+Depending on how the MUA resolves discrepancies between the Protected Headers and the Exposed Headers of a recieved message, these mailing list "features" may either break or the MUA may incorrectly interpret them as a security breach.
 
-FIXME: consider how to address this concern.
+Implementors may for this reason choose to implement slightly different strategies for resolving discrepancies, if a message is known to come from such a mailing list. Implementors should at the very least avoid "crying wolf" in such cases.
```
Comment (juga0) [inline draft-protected-headers.in.md]: I'd just avoid an idiom

```suggestion
Implementors may for this reason choose to implement slightly different strategies for resolving discrepancies, if a message is known to come from such a mailing list. MUAs should at the very least avoid presenting false alarms in such cases.
```

**Review by juga0 — APPROVED** [review]
I'm not familiar with the cases commented but make sense to me.
I'm just not sure about the non-capitalized may/should words. 
Just 2 comments on a typo and avoiding an idiom.

**Comment by dkg** [comment]
Sweet, thanks for all this, @BjarniRunar .  I've squashed these commits down to a single one (56697d8aeafb2eeb602686054fb573b38119be3f), so that "git blame" will not identify you as the source for some of the text that you added and removed in the back-and-forth discussion.  it's all merged now though :)

**Pull request closed without merging**